clusterbuster

```diff
@@ -2808,6 +2860,7 @@ function create_container_drop_cache() {
   image: "$container_image"
   securityContext:
     privileged: true
+    runAsUser: 0
```

Comment:
Why we need to add it runAsUser: 0 ?
Is it part of privileges issue ?

Reply:
Yes.  That's needed to run a privileged pod.